modules/nextflow/src/main/groovy/nextflow/executor/BashWrapperBuilder.groovy

```diff
@@ -461,6 +462,32 @@ class BashWrapperBuilder {
         }
     }
 
+    protected String getProcessDirectives() {
+        final lines = []
+        lines << '---'
+
+        if( arrayIndexName ) {
+            lines << 'array:'
+            lines << "  index-name: ${arrayIndexName}"
+            lines << "  index-start: ${arrayIndexStart}"
```

Comment:
I think fusion requires also the list of work directories requires by the child jobs